### ISSUE TITLE
Help with config using Windows Paths

### ISSUE THREAD

**Issue by doublee313** [knightzac19/PHP-ArkData#3]
Can you help me with this, I can't seem to get my Windows path working in PHP....

require_once("SourceQuery/bootstrap.php");
require_once("arkdata.php");
$arkdata = new Arkdata(array("installLocation"=>"D:\Servers\ARK\Servers\EEISLAND\ShooterGame\Saved\SavedArks"));
$arkdata->LoadSteam("xxxxxxxxxxxxxxxxxxxxxxxxx");
$arkdata->LoadOnlinePlayers();
$arkdataresults = $arkdata->getArkData();

I'm using IIS 10 w/ PHP 5.6 and it's working with other stuff....I'm just getting:

Invalid directory detected, no profiles or tribes found!

So I am assuming it's a stupid Windows issue with the path :(

Thanks,
Doublee

**Comment by knightzac19** [comment]
Have you tried using \\\\ instead of \\ ?

**Comment by doublee313** [comment]
Yes. Both seem to not like it. I wonder if IIS doesn’t like it being on a different drive. I can create a virtual directory for it and try it so I don’t have to switch drives. I will let you know.

By the way, are you still working on this?

Thanks!

**Comment by knightzac19** [comment]
Strange, considering you're using IIS, it's probably a permission issue at this point.

And no, I stopped working on this awhile ago because I converted it over to node.js which processed everything much faster and asynchronously.  https://github.com/knightzac19/NodeJS-ArkService

**Comment by doublee313** [comment]
Oh, do you recommend using that instead? I found some plugins for IIS for Node…. I’m just looking for something the Players can view for Player Info, Tribe Info, and maybe have some Admin utilities.

I just setup the 7Days Web Map which is pretty cool (has a few other features). You can check it out: http://7days.doubleegaming.com

Let me know what you think is best and I will go that route. Thanks for your time.

Also, I’m a beta testing for ASM 😊 Bletch is a great guy.

**Comment by knightzac19** [comment]
I'd definitely go for the node version instead (I've put many more man hours into that one), you can still use PHP on the frontend in IIS or a AJAX javascript frontend to go with it. There's really nothing you'd be able to do with IIS and Node.js directly, it's meant to be a standalone service that a web service connects to.

The map thing is pretty cool, one day someone will crack the code to player locations in the ark map save file and I'll probably release an update to get the coords for players from it. I've spent quite a few hours digging threw the binary of a map file without finding any reference to player locations.

As far as ASM goes, since Ark is finally going to be released to the masses, I might pick back up development on at least the Node.js project. There was just to many changes happening to the base of the server version for me to keep up with. I do have a side Angular project that I made that was supposed to be similar to ASM, but again, so many changes with Ark happened and I just didn't have the time, we'll see how the next few months goes.

**Comment by doublee313** [comment]
Thx man. Ill keep checking in and if yiu need help i can help test it.

Ill let you know my thoughts on this Node. Maybe ill have some ideas if yiu
dont mind :) I love help ASM and seeing it come to life.

Any better why to reach you?

**Comment by doublee313** [comment]
Zac,

Would you be kind enough to contact me at my email: doublee313@gmail.com and possibly remote into my server and see if you can get your Node working? I’m wasting a lot of my time since I lack the knowledge of these “plugins” and it’s prob something you can do in a few minutes. I’m also not stuck on using IIS if Apache or something else is going to be more compatible or easier. I’m just using IIS as it’s something I’ve had before. I’m used Apache before too so it’s not a huge learning curve.

Thanks,
Eddie
